hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystem.java

```diff
@@ -99,6 +99,7 @@
 public class TestOzoneFileSystem {
 
   private static final float TRASH_INTERVAL = 0.05f; // 3 seconds
+  private static final int listStatusBatchSize = 1024;
```

Comment:
I think we can omit this constant and rely on the default.  (It's also causing checkstyle failure.)